instrumentation/opentelemetry-instrumentation-psycopg2/tests/test_psycopg2_integration.py

```diff
@@ -22,15 +23,69 @@
 from opentelemetry.test.test_base import TestBase
 
 
+class MockCursor:
+
+    execute = mock.MagicMock(spec=types.MethodType)
+    execute.__name__ = "execute"
+
+    executemany = mock.MagicMock(spec=types.MethodType)
+    executemany.__name__ = "executemany"
+
+    callproc = mock.MagicMock(spec=types.MethodType)
+    callproc.__name__ = "callproc"
+
+    rowcount = "SomeRowCount"
+
+    def __init__(self, *args, **kwargs):
```

Comment:
Is this necessary?

Reply:
I think at some point the mock was a subclass of the real cursor so this was done to disable initializing the real cursor. Will remove.

Reply:
OK, this is actually required to satisfy the interface :) I forgot about that.